### ISSUE TITLE
registerTldrawLibraryVersion breaks Cloudflare Worker

### ISSUE THREAD

**Issue by peilingjiang** [tldraw/tldraw#4582]
### What happened?

The new version's operations at the global scope break the wrangler setup with the following error message (running `wrangler dev`).

```
 ⛅️ wrangler 3.78.8
-------------------

Your worker has access to the following bindings:
- Durable Objects:
  - TLDRAW_DURABLE_OBJECT: TldrawDurableObject
- R2 Buckets:
  - TLDRAW_BUCKET: tldraw-preview
⎔ Starting local server...
✘ [ERROR] service core:user:demo-sync: Uncaught Error: Disallowed operation called within global scope. Asynchronous I/O (ex: fetch() or connect()), setting a timeout, and generating random values are not allowed within global scope. To fix this error, perform this operation within a handler. https://developers.cloudflare.com/workers/runtime-apis/handlers/

    at null.<anonymous> (worker.js:8193:28) in registerTldrawLibraryVersion
    at null.<anonymous> (worker.js:9338:1)


✘ [ERROR] MiniflareCoreError [ERR_RUNTIME_FAILURE]: The Workers runtime failed to start. There is likely additional logging output above.
```

### How can we reproduce the bug?

Set up a Cloudflare sync example with the latest version of `@tldraw/tldraw` and `@tldraw/sync-core`.

### Where did this happen?

The developer package

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by SomeHats** [comment]
Thanks for the report! A fix for this will be in 3.1 which should be out very soon

**Comment by peilingjiang** [comment]
Resolved with v3.1.0

**Comment by kiastorm** [comment]
I'm still getting this with version 3.2.0
